pkg/controller/route/route_controller.go

```diff
@@ -62,19 +65,19 @@ type RouteController struct {
 	routes           cloudprovider.Routes
 	kubeClient       clientset.Interface
 	clusterName      string
-	clusterCIDR      *net.IPNet
+	clusterCIDRs     []*net.IPNet
```

Comment:
storing this as a map[family]IPNet would be clearer if/when we add multiple CIDRs for real

Reply:
The order matter because IPs gets assigned according to order of cidr. currently this is the *only* control user have over v4/v6 vs v6/v4